content/blog/2018-08-23-interview-with-presslabs.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Interview with Presslabs
+created_at: 2018-08-23
+kind: article
+author_name: Brian Brazil
+---
+
+*Continuing our series of interviews with users of Prometheus, Mile Rosu
+from Presslabs talks about their monitoring journey.*
+
+## Can you tell us about yourself and what Presslabs does?
+
+[Presslabs](https://www.presslabs.com/) is a high-performance managed WordPress
+hosting platform targeted at publishers, Enterprise brands and digital agencies
+which seek to offer a seamless experience to their website visitors, 100% of
+the time. 
+
+Recently, we have developed an innovative component to our core
+product—WordPress Business Intelligence. Users can now get real time,
```

Comment:
"Real time" in computers is usually hyphenated.

Reply:
Done